cypress-io/cypress
# fix: percy snapshots screenshotName should always be populated

- Closes UNIFY-1739

### User facing changelog
n/a

### Additional details
Before, when Percy snapshots were taken in the `afterEach` block, they all get the same name, resulting in the error below in CI.

### Steps to test
1. Look at Percy CI in this PR and confirm no errors in console for `SnapshotControls` component
2. Look at Percy CI in a different PR and confirm there are errors in the console for `SnapshotControls` component
3. Search through e2e and component tests in CircleCI to confirm there are no other Percy errors including `snapshot must be unique`
 
### How has the user experience changed?
**Before:**
<img width="1178" alt="Screen Shot 2022-05-27 at 8 28 39 AM" src="https://user-images.githubusercontent.com/26726429/170730933-837b5ca1-e6fa-4d4f-acba-c5bc4a8e4be7.png">

**After:**
<img width="964" alt="Screen Shot 2022-05-27 at 8 28 30 AM" src="https://user-images.githubusercontent.com/26726429/170731027-b4004726-96df-4014-9a93-84c364898ec3.png">


### PR Tasks


- [na] Have tests been added/updated? -- unfortunately there are no existing tests for this file, would be good to add tests for the whole file in a separate PR
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
